src/entities/Proposal/routes.ts

```diff
@@ -213,6 +207,35 @@ export async function createProposalPOI(req: WithAuth) {
   })
 }
 
+export async function createAddProposalPOI(req: WithAuth) {
+
+  const verify = async (config: NewProposalPOI) => {
```

Comment:
I like the idea of a verifying fn depending on the poi type.

Maybe if we decide to use a PoiType, it could decide how to validate internally.